src/main/kotlin/no/nav/familie/oppdrag/repository/MellomlagringKonsistensavstemmingRepository.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.familie.oppdrag.repository
+
+import no.nav.familie.oppdrag.service.Fagsystem
+import org.springframework.data.jdbc.repository.query.Query
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+interface MellomlagringKonsistensavstemmingRepository :
+    RepositoryInterface<MellomlagringKonsistensavstemming, UUID>,
+    InsertUpdateRepository<MellomlagringKonsistensavstemming> {
+
+    fun findAllByFagsystemAndAvstemmingstidspunktAndAktiv(
+        fagsystem: Fagsystem,
+        avstemmingstidspunkt: String,
+        aktiv: Boolean
+    ): List<MellomlagringKonsistensavstemming>
+
+    @Query(
+        "SELECT COALESCE(sum(antall_oppdrag),0) from mellomlagring_konsistensavstemming " +
+            "WHERE fagsystem = :fagsystem AND avstemmingstidspunkt = :avstemmingstidspunkt AND aktiv = true"
```

Comment:
Litt usikker på bruket av avstemmingstidspunkt her

Reply:
I praktiken skal COAELSCE ikke være nødvendig, men for å dekke alle teoretiske use case så må den være her. (startmelding + avsluttmelding med oppdrag). Derfor ønsker jeg å beholde COAELSCE.

Reply:
Varfør det? `COAELSCE` behøvs væl bara om `antall_oppdrag` skulle vara nullable? Vilket den inte ær

Reply:
Kolumnen er nullabdle men COALESH brukes om man ikke finner en rad

Reply:
Kolumnen ær `NOT NULL`
Hvis man ikke finner en rad så er det automatiskt 0 for sum?